src/App.jsx

```diff
@@ -1,10 +1,19 @@
 import './App.css';
 import Packing from './view/Packing';
+import Navbar from './component/Navbar';
+import { Routes, Route } from 'react-router-dom';
+import Budget from './view/Budget';
+import Blog from './view/Blog';
 
 function App() {
   return (
     <>
-      <Packing></Packing>
+      <Navbar />
```

Comment:
Good use of old react router, but I recommend you use the new version with outlet. This in a new way that Remix.run uses. 

It uses the component Outlet as a sort of conditional rendering, showing only the path to the view that you call. Here is a basic example

```
// Here you can declare a father route with its childrens, in the father route you will place both the navbar (that you want to show all the time) and the Outlet. The Outlet will show the component/view that you declared in children
const router = createBrowserRouter([
  {
    path: '/',
    element: <App />,
    errorElement: <div>Not Found</div>,
    children: [
      {
        path: '/',
        element: <Packing />,
      },
      {
        path: '/budget',
        element: <Budget />,
      },
      {
        path: '/blog',
        element: <Blog />,
      },
    ],
  },
]);

Reply:
All this goes in main, and in app you can have the navbar and the outlet